changelog.md

```diff
@@ -1,3 +1,25 @@
+# 0.6
+
+## Major changes
+
+* MAINT documentation (nearly every part was improved and extended, 
+  including installation, examples, API) 
+* ADD EPILS as mode (modified version of ParamILS)
+* MAINT minimal required versions of configspace, pyrfr, sklearn increased
+  (several issues fixed in new configspace version)
+* MAINT for quality scenarios, the user can specify the objective 
+  value for crashed runs
```

Comment:
This should be "FIX  #229, replace NaN and Inf with value for crashed runs" if that refers to this issue